nifi-extension-bundles/nifi-redis-bundle/nifi-redis-extensions/pom.xml

```diff
@@ -62,6 +62,17 @@
             <artifactId>jedis</artifactId>
             <version>${jedis.version}</version>
             <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.json</groupId>
+                    <artifactId>json</artifactId>
```

Comment:
This version of the `json` librar [20240303](https://central.sonatype.com/artifact/org.json/json/20240303) is actually licensed under the Public Domain, not the historically banned license. For that reason, the exclusion is not necessary and the banned dependency settings can be adjusted.